scripts/redirect_url_utils.py

```diff
@@ -25,11 +25,31 @@
         <meta http-equiv="REFRESH" content="0.0;url={redirect_to_url}" />
     </head>
     <body></body>
-</html>'''
+</html>
+'''
 
 def apply_map():
+  _add_redirects_from_file(FILE_TO_URL)
+
+def update_map(file):
+  """Update the main redirects file with the new redirect pairs in 'file'."""
+  if not os.path.isfile(file):
+    print("'%s' is not a file. Try again." % file)
+    return False
+  _add_redirects_from_file(file)
+  with open(FILE_TO_URL, 'r', encoding='utf-8') as f:
+    redirects = f.readlines()
+  with open(file, 'r', encoding='utf-8') as f:
+    new_redirects = f.readlines()
+  for new_re in new_redirects:
+    redirects.append(new_re)
+  with open(FILE_TO_URL, 'w', encoding='utf-8') as w:
+    for r in redirects:
+      w.write(r)
```

Comment:
nit:
Lines 40-48 - can we just read new redirects and append to an existing file? 
```
with open(file, 'r', encoding='utf-8') as f:
  new_redirects = f.readlines()
with open(FILE_TO_URL, 'a', encoding='utf-8') as w:
    for r in new_redirects:
        w.write(r)
```

Reply:
Yep, definitely better, thanks much